src/inmanta/app.py

```diff
@@ -293,6 +294,30 @@ def modules(options: argparse.Namespace) -> None:
     tool.execute(options.cmd, options)
 
 
+def generate_project_parser_config(parser: ArgumentParser) -> None:
+    parser.add_argument("--name", "-n", help="The name of the new project", required=True)
+    parser.add_argument("--output-dir", "-o", help="Output directory path", default="./")
+    parser.add_argument(
+        "--default", help="Use default parameters for the project generation", action="store_true", default=False
+    )
+
+
+@command(
+    name="generate-project", help_msg="Generate directory structure for a project", parser_config=generate_project_parser_config
```

Comment:
I think this should be a sub command of the project main command. 

@bart: wdyt?